src/core/sizage.rs

```diff
@@ -0,0 +1,7 @@
+#[derive(Debug, PartialEq)]
+pub struct Sizage {
```

Comment:
We will need to move this under matter, as Indexer also has Sizage (Xizage) in python